tools/remove_jiv_data_from_db_and_filesystem.pl

```diff
@@ -0,0 +1,152 @@
+#! /usr/bin/perl
+
+
+=pod
+
+=head1 NAME
+
+remove_jiv_data_from_db_and_filesystem.pl -- Cleans up the JIV data from the
+database tables and the filesystem
+
+=head1 SYNOPSIS
+
+perl remove_jiv_data_from_db_and_filesystem.pl C<[options]>
+
+Available option is:
+
+-profile: name of the config file in ../dicom-archive/.loris_mri
+
+=head1 DESCRIPTION
+
+This script will remove the JIV files from the C<parameter_file> table and
+move them to the C<$data_dir/archive/jiv> directory of the filesystem for
```

Comment:
it is not the `archive/jiv/` directory, you called it `archive/bkp_jiv_produced_before_LORIS_20.0//`